app/views/achievements/_form.html.haml

```diff
@@ -10,17 +10,17 @@
         = f.text_area :description, class: "form-control"
     .form-group
       = f.label :points, t(:achievement_points), class: ["col-sm-2", "control-label"]
-      .col-sm-10
-        = f.text_field :points, class: "form-control"
+      .col-sm-2
+        = f.text_field :points, class: "form-control", type: "number", min: 1, max: 3, size: 5
```

Comment:
Typed this as a `number` field which adds some easy client-side browser-based validation -- can only enter numbers -- and set an unsurprising range. If there are ever acheevs which result in negative points -- hee hee -- or acheevs that are worth more than 3 points then this range can be revisited easily enough.